### ISSUE TITLE
Make get_color more robust

### ISSUE THREAD

**Issue by astrojuanlu** [poliastro/czml3#42]
* If given a `Color`, return a `Color`
* If given a tuple, convert to list
* If the tuple contains things that *resemble* integers (i.e. numpy integers) do not crash

**Comment by astrojuanlu** [comment]
Partially addressed in #66

**Comment by Stoops-ML** [comment]
Closing as complete because as of v2.0.0 CZML3 classes use pydantic which converts applicable types to list. For example, creating a `Color` object with a numpy array, list and set:
```
import numpy as np
assert Color(rgba=[200, 100, 30]) == Color(rgba=[200, 100, 30]) == Color(rgba=(200, 100, 30))
```
